assets/docs/pages/operations/debug.md

```diff
@@ -0,0 +1,311 @@
+         namespace: {{< reuse "docs/snippets/namespace.md" >}}
+     hostnames:
+       - "debug.com"
+     rules:
+       - backendRefs:
+           - name: httpbin
+             port: 8000
+   EOF
+   ```
+
+5. Get the address of the debug gateway proxy.
+
+   * **Cloud Provider LoadBalancer**
+     ```sh
+     export INGRESS_GW_ADDRESS=$(kubectl get svc -n {{< reuse "docs/snippets/namespace.md" >}} debug-gateway -o=jsonpath="{.status.loadBalancer.ingress[0]['hostname','ip']}")
+     echo $INGRESS_GW_ADDRESS
+     ```
+   
+   * **Port-forward for local testing**
+     ```sh
+     kubectl port-forward deployment/debug-gateway -n {{< reuse "docs/snippets/namespace.md" >}} 8080:8080
+     ```
+
+6. Send traffic through the debug gateway proxy.
+
+   * **Cloud Provider LoadBalancer**
+     ```sh
+     curl -i http://$INGRESS_GW_ADDRESS:8080/headers -H "host: debug.com:8080"
+     ```
+   * **Port-forward for local testing**
+     ```sh
+     curl -i localhost:8080/headers -H "host: debug.com"
+     ```
+
+7. Review the debug logs for the gateway proxy.
+
+   ```sh
+   kubectl logs -n {{< reuse "docs/snippets/namespace.md" >}} -l app.kubernetes.io/name=debug-gateway
+   ```
+
+   Example output:
+
+   ```console
+   [2025-07-08 18:59:13.234][34][debug][pool] [external/envoy/source/common/conn_pool/conn_pool_base.cc:254] [Tags: "ConnectionId":"2"] destroying stream: 0 active remaining, readyForStream false, currentUnusedCapacity 1
+   [2025-07-08 18:59:14.240][34][debug][connection] [external/envoy/source/common/network/connection_impl.cc:774] [Tags: "ConnectionId":"1"] remote close
+   [2025-07-08 18:59:14.241][34][debug][connection] [external/envoy/source/common/network/connection_impl.cc:314] [Tags: "ConnectionId":"1"] closing socket: 0
+   [2025-07-08 18:59:14.243][34][debug][conn_handler] [external/envoy/source/common/listener_manager/active_stream_listener_base.cc:136] [Tags: "ConnectionId":"1"] adding to cleanup list
+   [2025-07-08 18:59:14.244][1][debug][main] [external/envoy/source/server/server.cc:245] flushing stats
+   [2025-07-08 18:59:18.232][34][debug][connection] [external/envoy/source/common/network/connection_impl.cc:774] [Tags: "ConnectionId":"2"] remote close
+   [2025-07-08 18:59:18.233][34][debug][connection] [external/envoy/source/common/network/connection_impl.cc:314] [Tags: "ConnectionId":"2"] closing socket: 0
+   [2025-07-08 18:59:18.233][34][debug][client] [external/envoy/source/common/http/codec_client.cc:107] [Tags: "ConnectionId":"2"] disconnect. resetting 0 pending requests
+   [2025-07-08 18:59:18.234][34][debug][pool] [external/envoy/source/common/conn_pool/conn_pool_base.cc:532] [Tags: "ConnectionId":"2"] client disconnected, failure reason: 
+   [2025-07-08 18:59:18.235][34][debug][pool] [external/envoy/source/common/conn_pool/conn_pool_base.cc:500] invoking 1 idle callback(s) - is_draining_for_deletion_=false
+   ```
+
+{{% /conditional-text %}}
+
+## Policy not applied {#trafficpolicy}
```

Comment:
since links are not working in agw and I am also not sure if gatewayextension is still used for agw, maybe we just make this envoy only for now?